packages/helpers/source/uri.test.ts

```diff
@@ -2,22 +2,23 @@ import { describe } from "@payvo/sdk-test";
 
 import { URI } from "./uri";
 
-describe("URI", async ({ assert, beforeEach, each, it, nock, loader }) => {
+describe("URI", async ({ assert, beforeEach, each, it }) => {
 	beforeEach(async (context) => (context.subject = new URI()));
 
 	it("should serialize", (context) => {
 		const result = context.subject.serialize({
-			method: "transfer",
+			amount: 1.2,
 			coin: "ark",
+			origin: "example.com",
```

Comment:
What is this parameter needed for?

Reply:
this is the hostname that needed to generate the new uri which pass from ww (didn't want to hardcode it)
we could also get the parameters and then combine it with the hostname in ww

Reply:
Remove it and decide outside the SDK which host should be used.

Reply:
fixed it